softwareconfiguration/mlplan/sklearn/resources/automl/searchmodels/sklearn/sklearn-rul.json

```diff
@@ -0,0 +1,53 @@
+{
+	"repository": "RUL",
+	"include": [
+		"./sklearn-rul-timeseries_transformer.json",
+		"./sklearn-rul-data_cleaner.json",
+		"./sklearn-preprocessors.json",
+		"./sklearn-regressors.json"
+	],
+	"components": [
+		{
+			"name": "sklearn.pipeline.make_pipeline",
+			"providedInterface": [
+				"MLPipeline",
+				"AbstractRegressor"
+			],
+			"requiredInterface": [
+				{
+					"id": "timeseries_transformer",
+					"name": "TimeseriesTransformer"
+				},
+				{
+					"id": "data_cleaner",
+					"name": "DataCleaner"
+				},
+				{
+					"id": "preprocessor0",

```

Comment:
Why do we have more than 1 preprocessor?

Reply:
Replaced search space to be up to date.